test/volumes-no-fabric-network.test.js

```diff
@@ -0,0 +1,113 @@
+    var CLIENT;
+    var SERVER;
+    var USER;
+
+    var testVolumeName =
+        common.createResourceName('test-volumes-no-fabric-network');
+
+    test('setup', function (t) {
+        common.setup({clientApiVersion: '~8.0'},
+            function onSetupDone(_, clients, server) {
+                CLIENTS = clients;
+                CLIENT = clients.user;
+                SERVER = server;
+
+                t.end();
+            });
+    });
+
+    /*
+     * This is necessary so that we proceed with the rest of the tests suite
+     * only after the entry for the newly added user is present in UFDS.
+     */
+    test('getting config from ufds', function (t) {
+        mod_testConfig.waitForAccountConfigReady(CLIENT,
+            function onConfigReady(configReadyErr, ufdsConfig) {
+                t.ifErr(configReadyErr, 'newly created user\'s config should ' +
+                    'eventually be created');
+                t.end();
+            });
+    });
+
+    test('getting user account from ufds', function (t) {
+        CLIENT.ufds.getUser(CLIENT.login, function onGetUser(getUserErr, user) {
+            USER = user;
+            t.end();
+        });
+    });
+
+    test('deleting user\'s dclocalconfig to remove default fabric network',
+        function (t) {
+            CLIENT.ufds.deleteDcLocalConfig(USER.uuid,
+                USER.dclocalconfig.dclocalconfig,
+                function onDcLocalCfgUpdated(err) {
+                    t.ifError(err, 'deleting dclocalconfig should succeed');
+                    t.end();
+                });
+        });
+
+    test('creating volume with default params should fail',
+        function (t) {
+            CLIENT.post('/my/volumes', {
+                name: testVolumeName,
+                type: 'tritonnfs'
+            }, function onVolumeCreated(volumeCreationErr, req, res, body) {
+                var expectedStatusCode = 409;
+                var expectedErrMsg =
+                    'default_network is not configured for account';
+                var expectedErrorCode =
+                    'DefaultFabricNetworkNotConfiguredError';
+
+                t.ok(volumeCreationErr,
+                    'creating a volume with default parameters should error');
+                t.equal(volumeCreationErr.statusCode, expectedStatusCode,
+                    'resposne status code should be ' + expectedStatusCode +
+                        ', was' + volumeCreationErr.statusCode);
+                t.equal(body.code, expectedErrorCode,
+                    'body\'s code should be ' + expectedErrorCode + ', was ' +
+                        body.code);
+                t.equal(body.message, expectedErrMsg,
+                    'body\'s message should be ' + expectedErrMsg + ', was ' +
+                        body.message);
+
+                t.end();
+            });
+    });
+
+    test('teardown', function (t) {
+        common.teardown(CLIENTS, SERVER, function (teardDownErr) {
```

Comment:
Is it intentional that this is `teardDownErr` instead of `tearDownErr` or `teardownErr`? I realize it matches below, it just seems like a typo to me. Maybe not!